scripts/build_db_from_esri.py

```diff
@@ -1072,7 +1072,38 @@ class ConversionMapping:
             8806: 'False_Easting',
             8807: 'False_Northing',
         }
-    )
+    ),
+    'Mercator': ConversionMapping(
+        epsg_code='9805',
+        epsg_name='Mercator (variant B)',
+        param_mapping={
+            8823: 'Standard_Parallel_1',
+            8802: 'Central_Meridian',
+            8806: 'False_Easting',
+            8807: 'False_Northing',
+        }
+    ),
+    'Equidistant_Conic': ConversionMapping(
+        epsg_code='9805',
```

Comment:
Why mapping Equidistant_Conic to Mercator variant B ? Doesn't look appropriate according to https://github.com/Esri/projection-engine-db-doc/blob/master/other/proj4-esri.txt#L182

Reply:
Hmmmm, not sure how that snuck in! 😱